tests/test_rabota_by.py

```diff
@@ -0,0 +1,45 @@
+"""This module contains tests for rabota.by pages executed with pytest library"""
+
+
+def test_rabota_by_connection(rabota_by_python_response):
+    assert rabota_by_python_response.status_code == 200, "The server does not respond"
```

Comment:
"The server does not respond" - from this message I don't understand what is going wrong if test fails

Reply:
Message has been changed.